.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "tabWidth": 2,
+  "useTabs": false,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "semi": false
+}
```

Comment:
Not great, but a potential start.

Reply:
I think this is fine for now. We usually aren't too prescriptive.